lib/iex/endpoints/ref_data.rb

```diff
@@ -2,7 +2,7 @@ module IEX
   module Endpoints
     module RefData
       def ref_data_isin(isins, options = {})
-        response = post('ref-data/isin', { token: publishable_token, isin: isins }.merge(options))
+        response = get('ref-data/isin', { token: publishable_token, isin: isins.join(',') }.merge(options))
```

Comment:
What do you think about adding a check to see if `isins` is indeed an array? If you pass an string for `isins` it will blow up

Reply:
I would `Array(isins)` to allow a single ASIN without having to do any work.